example/linux/main.go

```diff
@@ -56,38 +56,63 @@ func main() {
 	initrd := os.Getenv("INITRD_PATH")
 	diskPath := os.Getenv("DISKIMG_PATH")
 
-	bootLoader := vz.NewLinuxBootLoader(
+	bootLoader, err := vz.NewLinuxBootLoader(
 		vmlinuz,
 		vz.WithCommandLine(strings.Join(kernelCommandLineArguments, " ")),
 		vz.WithInitrd(initrd),
 	)
+	if err != nil {
+		panic(err)
```

Comment:
Could you replace panic with log.Fatal(err)?
I used panic because it does not complete the setup for logger in line 41.